brightcove-exoplayer/OfflinePlaybackSampleApp/src/main/java/com/brightcove/player/samples/offlineplayback/MainActivity.java

```diff
@@ -164,11 +171,15 @@ private void updateVideoList() {
             emptyListMessage.setText(R.string.fetching_playlist);
             emptyListMessage.setVisibility(View.VISIBLE);
 
-            playlist.findPlaylist(catalog, new PlaylistListener() {
+            HttpRequestConfig.Builder httpRequestConfigBuilder = new HttpRequestConfig.Builder();
+            httpRequestConfigBuilder.setBrightcoveAuthorizationToken(pasToken);
```

Comment:
are there any problems by passing a fake token? e.g. "YOUR_PAS_TOKEN"

Reply:
No. I've tested this change with an account not configured for PAS, and this token value is sent but ignored.